docs/guide/rest-quick-start.md

```diff
@@ -185,7 +186,11 @@ For example, the URL `http://localhost/users?fields=id,email` will only return t
 
 > Info: You may have noticed that the result of `http://localhost/users` includes some sensitive fields,
 > such as `password_hash`, `auth_key`. You certainly do not want these to appear in your API result.
-> You can and should filter out these fields as described in the [Resources](rest-resources.md) section.
+> You can and should remove these fields from result as described in the [Resources](rest-resources.md) section.
+
+Addionally, you can filter collections like `http://localhost/users?filter[id]=10` or
+`http://localhost/users?filter[email][like]=gmail.com` and sort collections like
+`http://localhost/users?sort=email` or `http://localhost/users?sort=-email`.
 
```

Comment:
should have a link to further reading here, link to `rest-resources.md#filtering-collections`